features/guidance/collapse.feature

```diff
@@ -725,10 +725,10 @@ Feature: Collapse
 
         When I route I should get
             | waypoints | route                 | turns                                      |
-            | a,g       | road,cross,cross      | depart,turn left,arrive                    |
-            | a,e       | road,road,road        | depart,continue straight,arrive            |
+            | a,g       | road,cross,cross      | depart,fork left,arrive                    |
+            | a,e       | road,road,road        | depart,fork slight right,arrive            |
```

Comment:
this test here was supposed to collapse the turns `bcg` and `bde`,`bdf` which now does not happen anymore.

This change modifies the purpose of the test.